lambda/src/main/java/com/lambda/IssueUpdater.java

```diff
@@ -21,6 +25,9 @@ public class IssueUpdater {
     private BacklogClient client;
 
     static private final String CUSTOM_FIELD_STARTED_AT = "Started at";
```

Comment:
Thank you for following the recommended accesser order, could you please change this line too?
```suggestion
    private static final String CUSTOM_FIELD_STARTED_AT = "Started at";
```